docs/design/coreclr/jit/lsra-detail.md

```diff
@@ -567,16 +589,15 @@ LinearScanAllocation(List<RefPosition> refPositions)
         -   Whether the `RefPosition` being allocated, or the one
             potentially being spilled, is reg-optional
 
+    -   Both `tryAllocateFreeReg()` and `allocateBusyReg()` currently fully evaluate the "goodness"
+        of each  register.
+
     -   It will always spill an `Interval` either at its most recent
         use, or at the entry to the current block.
 
-        -   Issues [\#7609](https://github.com/dotnet/coreclr/issues/7609) and
```

Comment:
Any reason for deleting the pointer to the tracking issues?

Reply:
Ah, I see that you have moved them down. Never mind.

---
In reply to: [457638736](https://github.com/dotnet/runtime/pull/39492#discussion_r457638736) [](ancestors = 457638736)